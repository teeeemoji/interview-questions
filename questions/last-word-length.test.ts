import lastWordLength from './last-word-length'

test('test last word length from a string', () => {
    expect(lastWordLength('I am baby')).toMatchSnapshot()
})

test('test last word length from a string with symbol', () => {
    expect(lastWordLength('Don\'t say u1s1!')).toMatchSnapshot()
})