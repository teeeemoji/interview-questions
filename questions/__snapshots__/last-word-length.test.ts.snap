// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test last word length from a string 1`] = `4`;

exports[`test last word length from a string with symbol 1`] = `5`;
